src/render/components/Dom.jsx

```diff
@@ -33,10 +33,9 @@ const DOM = props => {
 		appMarkup = '',
 		assetPublicPath,
 		baseUrl,
-		clientFilename,
 		head,
 		initialState = {},
-		scripts = [],
+		scripts,
```

Comment:
definitely, which is a good thing - there's no way to gracefully handle that kind of programmer error, so best to crash hard.